projects/go-lib/src/lib/components/go-table/go-table.component.ts

```diff
@@ -60,10 +61,11 @@ export class GoTableComponent implements OnInit, OnChanges {
     return false;
   }
 
-  columnClasses(columnField: string) : object {
-    return {
-      'go-table__th--sort-down': this.sortClasses(columnField, SortDirection.ascending),
-      'go-table__th--sort-up': this.sortClasses(columnField, SortDirection.descending)
+  sortIcons(columnField: string) : string {
```

Comment:
Can we please declare whether all the functions in this file are `public` or `private`? Seems like a good practice to get into. Let's plan on bringing this choice to the group next time we meet up and make a decision.

Reply:
Sounds fair enough to me.